### PR TITLE
Fix login bug

### PR DESCRIPTION
**Pull request by williamcabrera4**
-When we load the Unleash site we can click to login button with a current logged session and show the account select dialog.

### CODE REVIEW

**Comment by williamcabrera4** [comment]
Before:
bug

**Comment by williamcabrera4** [comment]
Before:
bug2

**Comment by williamcabrera4** [comment]
After:
fix2

File: app/services/authService.js [hunk 1]
```diff
@@ -26,6 +26,7 @@ class AuthService {
     if (user) {
       const userProviderData = _.head(user.providerData);
       const userId = userProviderData.uid;
+      this.dispatch(UserActions.showUserFetchSpinner());
```
Comment (szymonmichalak) [inline app/services/authService.js]: What if user login action fails? Spinner will remain ON.

I'm also not sure if that's the right way to describe what's going on here ;-) Action is called `USER.FETCH.SHOW_SPINNER`, which is triggered in `authService.js`. Let's be consistent here and do actions like:
1. `USER.AUTH.LOGIN_START` - `isLoading` flag is set to `true`,
2. `USER.AUTH.LOGIN` -->`USER.AUTH.LOGIN_SUCCESS`,
3.  `USER.AUTH.LOGIN_FAILURE` - `isLoading` flag is set to `false` + maybe some error?

**Comment by williamcabrera4** [comment]
@szymonmichalak updated ;)

**Comment by szymonmichalak** [comment]
Looks good @williamcabrera4! 👍

**Pull request merged**